### PR TITLE
docs: add Refine site to showcase

### PR DESCRIPTION
**Pull request by omeraplak**
## Motivation

The documentation of Refine is using Docusaurus v2. It could be cool to add it in the showcase page.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Run yarn run start and check that the page http://localhost:3000/showcase is rendered properly.

## Related PRs

No

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Hi @omeraplak! Have you managed to sign the CLA?

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1280,6 +1280,14 @@ const Users: User[] = [
     source: 'https://github.com/redis-developer/redis-developer.github.io',
     tags: ['opensource', 'product', 'design', 'favorite'],
   },
+  {
+    title: 'Refine',
+    description: 'A React-based framework for building data-intensive applications in no time!',
+    preview: require('./showcase/refine.png'),
+    website: 'https://refine.dev',
+    source: 'https://github.com/pankod/refine',
+    tags: ['opensource', 'product', 'design', 'versioning', 'large'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1289]:
```suggestion
    tags: ['favorite', 'opensource', 'product', 'design', 'versioning', 'large'],
```

**Comment by Josh-Cena** [comment]
I can't commit to your branch, but I'd like your site to be added to the favorites! Nice site

**Comment by omeraplak** [comment]
> Hi @omeraplak! Have you managed to sign the CLA?

Hey,
I've already CLA but seems like it a not signed. I sent a email to "cla@fb.com". I hope we can pass the CLA process soon. 

By the way, thanks for adding it to your favourites!

**Comment by Josh-Cena** [comment]
If you have waited for like 4 hours, then sure, otherwise we can wait a big longer😉 The CLA bot is quite laggy these days

**Comment by omeraplak** [comment]
> If you have waited for like 4 hours, then sure, otherwise we can wait a big longer😉 The CLA bot is quite laggy these days

Ohh! It's approved!

**Comment by Josh-Cena** [comment]
Great, let's go!

**Pull request merged**